web/src/components/nomination/LocationInput.vue

```diff
@@ -25,7 +25,7 @@
         :label="$t('nominateSpeaker.prefecture')"
         :error-messages="prefectureErrors"
         :items="prefectures"
-        :item-text="(e) => (e.get('prefecture'))"
+        :item-text="(e) => e.fields.prefecture"
```

Comment:
Using `record.get('field_name')` no longer works after our Airtable object goes through the lambda function and gets stringified into a response for our API service. I think this could be fixed by adding a serialization/deserialization step to our API requests, but I thought it's probably not worth the trouble right now, especially with the plan to migrate away from Airtable and to a proper backend in the near future.